src/init/CliRunner.ts

```diff
@@ -1,72 +1,70 @@
-import * as Path from 'path';
+import * as path from 'path';
 import type { ReadStream, WriteStream } from 'tty';
 import type { LoaderProperties } from 'componentsjs';
 import { Loader } from 'componentsjs';
 import yargs from 'yargs';
 import { getLoggerFor } from '../logging/LogUtil';
+import { ensureTrailingSlash } from '../util/PathUtil';
 import type { Setup } from './Setup';
 
 const logger = getLoggerFor('CliRunner');
 
 /**
  * Generic run function for starting the server from a given config
  * @param args - Command line arguments.
- * @param stdin - Standard input stream.
- * @param stdout - Standard output stream.
  * @param stderr - Standard error stream.
  * @param properties - Components loader properties.
  */
-export const runCustom = function(
-  args: string[],
-  stdin: ReadStream,
-  stdout: WriteStream,
-  stderr: WriteStream,
-  properties: LoaderProperties,
-): void {
-  const { argv } = yargs
+export const runCli = function({
+  argv = process.argv,
+  stderr = process.stderr,
+  properties = {
+    mainModulePath: path.join(__dirname, '../../'),
+  },
```

Comment:
Never seen this before, so we have variables, followed by defaults, followed by typings, followed by another default? :D

Reply:
All the way 😎